css/src/components/component.md

```diff
@@ -0,0 +1,8 @@
+#### Components
+
+Create new component when you have a complex layout that requires many helper classes to recreate. Keep following in mind while creating a New component .
+
+- Component must be reusable.
+- Should be output producing.
```

Comment:
I'm not entirely sure what this line is supposed to mean. Can it be reworded to something more simple?

Reply:
We can probably remove 
```
- Component must be reusable.
- Should be output producing.